cal/cal/views.py

```diff
@@ -36,7 +36,6 @@ def tag_detail(request, pk):
     try:
         tag = Tag.objects.get(user=request.user, id=pk)
         context['tag'] = tag
-        context['events'] = tag.query()
```

Comment:
See above comment, I thought it's better style to do this in the controller instead.